specification/resources/resource-manager/Microsoft.Resources/stable/2016-06-01/subscriptions.json

```diff
@@ -61,9 +61,6 @@
               "$ref": "#/definitions/LocationListResult"
             }
           }
-        },
```

Comment:
I strongly NOT suggest this change. This is a list operation, with a return type that follows the "value array" pattern. SDKs will be way better WITH the pageable.

Reply:
@vivsriaus Can you please comment?

Reply:
@lmazuel the /locations API does return a value array containing a list of locations, but on the service side, this is not pageable. I believe that is the reason why we initially added x-ms-pageable but set the nextLink to null. However, @shahabhijeet says it should be null because autorest ends up generating a separate Page1 class when nextLink:null is included. So, should the fix be in autorest (which I what I originally suggested), but @shahabhijeet disagrees. Thoughts?

Reply:
@vivsriaus first of all, I never suggested you should remove x-ms-pagable.
As a matter of fact, removing it will introduced a breaking change.

That being said, do you think this was an oversight when this particular list operation was modeled as x-ms-pagable with nextlink set to null when the service does not behave as a OData pagable?
I think leaving as is does not hurt.